presto-main/src/main/java/io/prestosql/execution/DynamicFilterConfig.java

```diff
@@ -37,19 +37,19 @@
     private boolean enableLargeDynamicFilters;
     private int serviceThreadCount = 2;
 
-    private int smallBroadcastMaxDistinctValuesPerDriver = 100;
-    private DataSize smallBroadcastMaxSizePerDriver = DataSize.of(10, KILOBYTE);
-    private int smallBroadcastRangeRowLimitPerDriver;
-    private int smallPartitionedMaxDistinctValuesPerDriver = 10;
+    private int smallBroadcastMaxDistinctValuesPerDriver = 200;
+    private DataSize smallBroadcastMaxSizePerDriver = DataSize.of(20, KILOBYTE);
+    private int smallBroadcastRangeRowLimitPerDriver = 5_000;
```

Comment:
0 -> ... is a big change. intentional?

(same for others)

Reply:
Yes. Additional CPU overhead is negligible, but we can support larger dimensions (E.g time or date dimensions)